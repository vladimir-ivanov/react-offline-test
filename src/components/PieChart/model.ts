export interface SeriesItem {
    id: string;
    label: string;
    value: number;
    color?: string;
}
