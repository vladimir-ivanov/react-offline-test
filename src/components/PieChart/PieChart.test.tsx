import '../../setupEnzyme';

import * as React from 'react';
import { mount } from 'enzyme';
import { PieChart } from './PieChart';

const fixture = [
    {
        label: 'one',
        id: 'one',
        value: 12
    },
    {
        label: 'two',
        id: 'two',
        value: 78
    }
];

//TODO - snapshot serialization needs fixing, does not work correctly
describe('PieChart', () => {
    it('should render correctly', () => {
        expect(mount(<PieChart data={fixture} />)).toMatchSnapshot();
    });
});
