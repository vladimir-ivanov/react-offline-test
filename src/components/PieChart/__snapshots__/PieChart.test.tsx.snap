// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PieChart should render correctly 1`] = `ReactWrapper {}`;
